domains/transfer/Transfer.API/API/ClaimAutomation/Api/Controllers/ClaimAutomationController.cs

```diff
@@ -14,21 +15,23 @@
 
 namespace API.ClaimAutomation.Api.Controllers;
 
-[Authorize]
 [ApiController]
-[ApiVersion(ApiVersions.Version20240103)]
+[Authorize(Policy.Frontend)]
+[ApiVersion(ApiVersions.Version20240515)]
 [Route("api/claim-automation")]
-public class ClaimAutomationController(IUnitOfWork UnitOfWork) : ControllerBase
+public class ClaimAutomationController(IUnitOfWork unitOfWork, AccessDescriptor accessDescriptor) : ControllerBase
```

Comment:
Did we forget to migrate this ?

Reply:
No. This is just how git shows diffs when you:

1. have two files named A and B.
2. Delete A
3. Rename B to A

Then it looks like what you see now.